app/routes/verkiezingen/verkiezingsuitslag/prepare.js

```diff
@@ -1,9 +1,60 @@
 import Route from '@ember/routing/route';
 
+import { service } from '@ember/service';
+
 export default class PrepareInstallatievergaderingRoute extends Route {
-  async model() {
+  @service store;
+
+  queryParams = {
+    filter: { refreshModel: true },
+    sort: { refreshModel: true },
+  };
+
+  async model(params) {
+    const verkiezingsuitslag = this.modelFor('verkiezingen.verkiezingsuitslag');
```

Comment:
Personally I would change this to parentModel, this is what we mostly use in other routes and then it is clear that this can contain multiple resources.